src/Engines/CollectionEngine.php

```diff
@@ -205,10 +205,12 @@ function ($row) use ($column, $keyword) {
      */
     public function paging()
     {
-        $this->collection = $this->collection->slice(
-            $this->request['start'],
-            (int) $this->request['length'] > 0 ? $this->request['length'] : 10
-        );
+        if(!$this->rewriteTotals) {
```

Comment:
Hey @yajra 
I think we need this in the live code.
Look at #706 for more information